jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/NetworkConnector.java

```diff
@@ -24,6 +25,7 @@ public interface NetworkConnector extends Connector, Closeable
     /**
      * <p>Performs the activities needed to open the network communication
      * (for example, to start accepting incoming network connections).</p>
+     * <p>Implementation must be idempotent.</p>
```

Comment:
Mustn't `close()` be idempotent too?

Reply:
Added javadocs to `close()` too, although it is more natural for close to automatically be idempotent.